web-app/js/ala-image-viewer.js

```diff
@@ -3,11 +3,18 @@ var imgvwr = {};
 (function(lib) {
 
     var map_registry = {};
+    var imageServiceBaseUrl = "http://dev.ala.org.au:8080/ala-images";
 
     var base_options = {
-        imageServiceBaseUrl: "http://images.ala.org.au",
+        imageServiceBaseUrl:  "http://dev.ala.org.au:8080/ala-images",
         auxDataUrl: '',
-        initialZoom: 'auto'
+        auxDataTitle: 'View more information about this image',
+        initialZoom: 'auto',
+        addDownloadButton: true,
```

Comment:
IMO all these extra options should be disabled by default.
The README.md file should have clear documentation about what initialization options are available.